k6/consistency/scripts/run-case.sh

```
@@ -0,0 +1,478 @@
+  --property print.value=true
+EOF
+)
+
+docker run --rm --network "${DOCKER_NETWORK}" "${KAFKA_IMAGE}" \
+  sh -lc "${OBSERVER_CMD}" >"${EVENT_LOG}" 2>&1 &
+OBSERVER_PID=$!
+
+sleep 2
+
+echo "[run-case] starting k6 load..."
+TEST_START_EPOCH="$(date +%s)"
```

Shellcheck (0.11.0)

[warning] TEST_START_EPOCH appears unused. Verify use (or export if used externally).

(SC2034)